pytorch_lightning/utilities/__init__.py

```diff
@@ -60,3 +60,17 @@ def _module_available(module_path: str) -> bool:
 class AMPType(Enum):
     APEX = 'apex'
     NATIVE = 'native'
+
+
+class DistributedType(Enum):
```

Comment:
```suggestion
class DistributedType(str, Enum):
```

If you make string enums inherit from str, you can do:

```
DistributedType.DP == "dp"
```

which is very convenient to avoid casting from/to str. Same for the other string enums in this file